src/+otp/+campbellmooredae/mass.m

```diff
@@ -0,0 +1,5 @@
+function M = mass(~, ~, ~, ~)
+
+M = sparse([9,10,7,8],[1,3,4,6],[-1,-1,1,1],10,10);
```

Comment:
I know that this notation is very efficient, but again for this low amount of variables, are sparse matrices still worth it?